### PR TITLE
Added a java replacement of jsonplaceholder.

### PR DESCRIPTION
**Pull request by 0x3D**
This is written using Spring boot IO and psotgressql. At the moment it
just copies the functionality that json server provided earlier but it
is very possible to extend. At the moment there isn't any documentation
for the code but it is in the works. To get postgres to play nice you
have to run the `tables.sql` in PSQL this can be done with `\i
tables.sql`. There is also a CLEAN file which drops *ALL* tables so be
careful that it is run in the right database. I have also written
inserts to get basic data up and running. At the moment I have no data
for the bookings but that can be written too. We should create a
setup.sql file to get up and running quickly.
If something is unclear please do not hesitate to contact me. Write on
Slack or comment on this commit.

### CODE REVIEW

**Review by erikbengtssonchalmers — APPROVED** [review]
Very nice, like it!!!

**Review by jonasn-chalmers — APPROVED** [review]
Så underbart å se lite java! Gött tryckt ser asfint ut!

File: javaserver/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        URL website = new URL(urlString);
+        ReadableByteChannel rbc;
+        rbc = Channels.newChannel(website.openStream());
+        FileOutputStream fos = new FileOutputStream(destination);
+        fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
+        fos.close();
+        rbc.close();
+    }
+
+}
```
Comment (axelhertzberg) [inline javaserver/.mvn/wrapper/MavenWrapperDownloader.java:117]: Vad används denna filen för?

File: javaserver/inserts.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO facilities VALUES ('Tvättstuga');
+INSERT INTO facilities VALUES ('Gym');
+INSERT INTO facilities VALUES ('Lokaler');
```
Comment (axelhertzberg) [inline javaserver/inserts.sql:3]: Vill jag helst ha alla users som inserts också, jag kan fixa det om du vil @0x3D

File: javaserver/pom.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+			<version>20210307</version>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
```
Comment (axelhertzberg) [inline javaserver/pom.xml:58]:
```suggestion
			</plugin>
			<plugin>
				<groupId>org.codehaus.mojo</groupId>
				<artifactId>exec-maven-plugin</artifactId>
				<version>1.4.0</version>
				<configuration>
					 <mainClass>com.msdos.msdosbooking.MsDosBookingApplication</mainClass>
				</configuration>
			</plugin>
```

Skulle gärna vilja ha detta pluginet också.

File: javaserver/inserts.sql (hunk 2 above)
Comment (axelhertzberg) [reply javaserver/inserts.sql:3]: Jag är efterbliven, såg dem inte tools mappen!

**Review by axelhertzberg — APPROVED** [review]
Jäkla dunder jobb!! Om du inte har något emot det får du gärna inkludera det pluginet. Gör det enklare att starta maven projectet på Unix.

File: javaserver/.mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (0x3D) [reply javaserver/.mvn/wrapper/MavenWrapperDownloader.java:117]: It is with Spring-boot. It downloads the mvnwrapper jar file. If you don't want to use it regular mvn works too.

File: javaserver/pom.xml (hunk 3 above)
Comment (0x3D) [reply javaserver/pom.xml:58]: I think this is not necessary when running with `mvn spring-boot:run` or `./mvnw spring-boot:run` if you want to use the wrapper.

File: javaserver/inserts.sql (hunk 2 above)
Comment (0x3D) [reply javaserver/inserts.sql:3]: Added a setup.sql so it is easier to setup database!

**Pull request merged**